poster/poster.tex

```diff
@@ -0,0 +1,367 @@
+ \end{itemize}
+ \item Each set of four rotated pictures were either in the training or testing set. 
+  \item Workflow of this project is shown as a flowchart in Figure~\ref{fig:flowchart}
+ \end{itemize}
+
+
+  \end{block} %tools and methods block
+
+\end{column}
+
+    \begin{column}{\sepwid}\end{column}			% empty spacer column
+
+
+        \begin{column}{\onecolwid}
+
+          
+ \begin{block}{Workflow}
+
+\tikzstyle{block} = [rectangle, draw, fill=SFUred!20, 
+    text width=30em, text centered, rounded corners, minimum height=2em]
+    \tikzstyle{small} = [rectangle, draw, fill=white!20, 
+    text width=5em, text centered, rounded corners, minimum height=2em]
+        \tikzstyle{medium} = [rectangle, draw, fill=white!20, 
+    text width=8em, text centered, rounded corners, minimum height=2em]
+
+\tikzstyle{line} = [draw, -latex']
+
+\begin{figure}   
+\centering 
+\begin{tikzpicture}[node distance = 4cm, auto]
+    % Place nodes
+    \node [block] (capture) {Capture images};
+    \node [block, below of=capture] (process) {Label and process images (crop, resize and convert to RGB)};
+%    \node [block, below of=process] (convert) {Convert to RGB };
+    \node [block, below of= process] (cnn) {Extract features via CNN (Caffe and AlexNet)};
+    \node [block, below of=cnn] (svm) {Classify data via SVM (SciKit Learn)};
+    \node [medium, below left = 2cm and -12cm of svm] (nonbanana) {Non-Banana};
+    \node [small, below right = 2cm and -14 cm of svm] (banana) {Banana};
+    \node [small, below of = banana] (ripe) {Ripe};
+    \node [small, below left = 1.8 cm and 1.8 cm of banana] (unripe) {Unripe};
+     \node [small, below right = 1.8cm and 1.8 cm of banana] (overripe) {Overripe};
+
+    % Draw edges
+    \path [line] (capture) -- (process);
+    \path [line] (process) -- (cnn);
+%    \path [line] (convert)--(cnn);
+    \path [line] (cnn)--(svm);
+    \path [line] (svm)--(nonbanana);
+        \path [line] (svm)--(banana);
+          \path [line] (banana)--(unripe);
+        \path [line] (banana)--(overripe);
+	\path [line] (banana)--(ripe);
+\end{tikzpicture}
+\caption{A flow chart of project development.}
+\label{fig:flowchart}
+ \end{figure}
+  \end{block} %tools and methods block
+         
+%         \begin{theorem}\label{linearHamiltonianTheorem}
+\vskip1ex
+
+\begin{alertblock}{Features Extraction via AlexNet}
+
+\begin{figure}[h]
+\centering
+\includegraphics[width=\textwidth]{alexnet.png}
+\caption{Architechture of AlexNet.}
+\end{figure}
+\end{alertblock}
+\vskip1ex
+\begin{block}{Results}
+
+\begin{table}[h]
+\caption{Overall percentage of correctly classified objects from training and testing of SVM models with various kernels. Features were obtained from FC6, FC7 and FC8 exits of AlexNet. (Lin = linear, RBF = radial basis function, Sig = sigmoid, Poly = polynomial)}
+\begin{tabular}{|c|c|c|c|c|c|c|c|c|}\hline
+ &  \multicolumn{4}{c|}{Training Results} & \multicolumn{4}{c|}{Testing Results}\\\hline
```

Comment:
I think this is the most important results and should be at the top right of the poster, I dunno we can discuss it today.